### DISCUSSION TITLE
Welcome to pico-infonesPlus Discussions!

### DISCUSSION THREAD

**Discussion by fhoedemakers** [fhoedemakers/pico-infonesPlus#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Brentos1989** [comment]
Hi just wondering if u could help i have made this project with the pcb and when i start it up i get a sd card mount fail error 1 , i have tried other sd cards and i get error 3 is there somthing i am missing?

**Comment by fhoedemakers** [comment]
Hi,

- First of all, check your soldering. Bad soldered connections to the SD-card cause those problems. See also #31
- Your SD card must be FAT32 formatted.
- Use low capacity SD-cards. 4, 16 or 32 GB should work. I recently bought [these](https://a.co/d/4dIhWtu) which work well. Keep away from cheap unbranded cards.

Frank

**Reply by fhoedemakers** [reply]
Did you manage to solve your issue?

**Comment by Brentos1989** [comment]
Hi mate 

 

Yes turns out it was low power 

 

Regards 

 

Brent Talarico

 Project Manager

 Vetro Architectural Glass & Aluminium Pty Ltd

 Phone: (02) 4955 3344

 Mobile: 0433 665 711

 Fax: (02) 4955 3343

**Comment by n2hyo** [comment]
Ok i am having this same issue with my build, i have tried a few 32g sandisk cards and either get an error 3 with not  inserted and error 13 with a card inserted. updated the firmware and tried a different power supply.

Thoughts

**Reply by fhoedemakers** [reply]
Assume you have the PCB. Please send a picture of the front and back.

**Comment by n2hyo** [comment]
Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: Frank Hoedemakers ***@***.***>
Date: 10/19/24 2:31 PM (GMT-05:00)
To: fhoedemakers/pico-infonesPlus ***@***.***>
Cc: "mike n2hyo.net" ***@***.***>, Comment ***@***.***>
Subject: Re: [fhoedemakers/pico-infonesPlus] Welcome to pico-infonesPlus Discussions! (Discussion #28)


Assume you have the PCB. Please send a picture of the front and back.

**Comment by fhoedemakers** [comment]
See no pictures

**Reply by n2hyo** [reply]
![20241019_144800](https://github.com/user-attachments/assets/93854907-1b00-4267-999e-f622269cb0ae)
![20241019_144811](https://github.com/user-attachments/assets/b0cf35b1-c4e4-4df3-befc-bc2e72c0d290)

**Comment by fhoedemakers** [comment]
It seems like the three debug holes on the right of the pico are not soldered. The debug middle hole is used for GND and is connected to GND of the SD breakout.
Also redo the solder work on the other pins of the pico. 

If you have a multimeter at hand, you can check continuity between the SD breakout board and the pico. How they are connected is described here:

https://github.com/fhoedemakers/pico-infonesPlus#pinout-1

**Reply by n2hyo** [reply]
That was it, i have never soldered any of the debug holes on any other projects but that fixed it.

Thank you for you help.

**Comment by fhoedemakers** [comment]
Glad to hear it's working now. Have fun!
Frank

**Comment by chrisvanriper** [comment]
I'm excited to try this with my 10 year old. We wanted to build a handheld- do you have it worked out, or have an idea of how we could directly run a little OLED or LCD screen like the GameBoy Pico projects do instead of the DVI breakout board? Would microswitches be hard to use instead of the NES controller ports? Either way, this is inexpensive enough that we may make one just to play with, then experiment with the other version. We want a gameboy color emulator for pico next!

**Reply by fhoedemakers** [reply]
> I'm excited to try this with my 10 year old. We wanted to build a handheld- do you have it worked out, or have an idea of how we could directly run a little OLED or LCD screen like the GameBoy Pico projects do instead of the DVI breakout board? Would microswitches be hard to use instead of the NES controller ports? Either way, this is inexpensive enough that we may make one just to play with, then experiment with the other version. We want a gameboy color emulator for pico next!

It shoud be possible. Last year a made a version of the emulator for the PicoSystem handheld. See https://github.com/fhoedemakers/PicoSystem_InfoNes 
Maybe that can be used as a start.  It basically uses microswitches as buttons. The most hardest part is finding the correct screen which is fast enough to display the data at 60fps. The PicoSystem has that kind of screen and has also a vsync setting to keep the framerate steady. I use the software library they provide to render the screen.

More info abot the PicoSystem: https://shop.pimoroni.com/products/picosystem?variant=32369546985555
